src/omotes_simulator_core/adapter/transforms/esdl_asset_mapper.py

```diff
@@ -47,20 +47,22 @@ class EsdlAssetMapper:
         esdl.HeatPump: HeatPump,
     }
 
-    def to_esdl(self, entity: AssetAbstract) -> Any:
+    @staticmethod
+    def to_esdl(entity: AssetAbstract) -> Any:
         """Maps entity object to PyEsdl objects."""
         raise NotImplementedError("EsdlAssetMapper.to_esdl()")
 
-    def to_entity(self, model: EsdlAssetObject) -> AssetAbstract:
+    @staticmethod
+    def to_entity(model: EsdlAssetObject) -> AssetAbstract:
         """Method to map an esdl asset to an asset entity class.
 
         :param EsdlAssetObject model: Object to be converted to an asset entity.
 
-        :return: Entity object.
+        :return: Entity object of type AssetAbstract.
         """
-        if not type(model.esdl_asset) in self.conversion_dict:
+        if not type(model.esdl_asset) in EsdlAssetMapper.CONVERSION_DICT:
```

Comment:
Move CONVERSION_DICT outside of EsdlAssetMapper.

Reply:
I have moved it. Please review it.